graphql/schema/types/dvd.graphql

```diff
@@ -0,0 +1,51 @@
+type Dvd {
+  id: ID!
+  checksum: String!
+  name: String!
+  aliases: String
+  durationdvd: String
```

Comment:
Is there a reason this is not named `duration`?

Reply:
I used "durationdvd" because I had a problem with the "duration" field in the [scenes] table. There is a "left join" in a query what caused me this problem.

Reply:
That can be resolved by adding the table qualifier to the query - `scenes.duration`, `dvd.duration`, for example.